### ISSUE TITLE
xarray attributeError while opening gribFile

### ISSUE THREAD

**Issue by meteoDaniel** [ecmwf/cfgrib#26]
I am getting the following Attribute Error when i am using: 

```
from cfgrib import xarray_store
ds=xarray_store.open_dataset('icon-eu_europe_regular-lat-lon_single-level_2018050300_000_ASOB_S.grib2')
```

Error:
```
In [3]: ds=xarray_store.open_dataset('icon-eu_europe_regular-lat-lon_single-level_2018050300_000_ASOB_S.grib2')
---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
<ipython-input-3-23c9c469eeaa> in <module>()
----> 1 ds=xarray_store.open_dataset('icon-eu_europe_regular-lat-lon_single-level_2018050300_000_ASOB_S.grib2')

/usr/local/lib/python3.6/dist-packages/cfgrib/xarray_store.py in open_dataset(path, backend_kwargs, filter_by_keys, **kwargs)
     46     real_backend_kwargs.update(backend_kwargs)
     47     store = cfgrib_.CfGribDataStore(path, **real_backend_kwargs)
---> 48     return xr.backends.api.open_dataset(store, **kwargs)
     49 
     50 

/usr/lib/python3/dist-packages/xarray/backends/api.py in open_dataset(filename_or_obj, group, decode_cf, mask_and_scale, decode_times, autoclose, concat_characters, decode_coords, engine, chunks, lock, cache, drop_variables)
    324         store = backends.ScipyDataStore(filename_or_obj)
    325 
--> 326     return maybe_decode_store(store)
    327 
    328 

/usr/lib/python3/dist-packages/xarray/backends/api.py in maybe_decode_store(store, lock)
    236             store, mask_and_scale=mask_and_scale, decode_times=decode_times,
    237             concat_characters=concat_characters, decode_coords=decode_coords,
--> 238             drop_variables=drop_variables)
    239 
    240         _protect_dataset_variables_inplace(ds, cache)

/usr/lib/python3/dist-packages/xarray/conventions.py in decode_cf(obj, concat_characters, mask_and_scale, decode_times, decode_coords, drop_variables)
    601     vars, attrs, coord_names = decode_cf_variables(
    602         vars, attrs, concat_characters, mask_and_scale, decode_times,
--> 603         decode_coords, drop_variables=drop_variables)
    604     ds = Dataset(vars, attrs=attrs)
    605     ds = ds.set_coords(coord_names.union(extra_coords).intersection(vars))

/usr/lib/python3/dist-packages/xarray/conventions.py in decode_cf_variables(variables, attributes, concat_characters, mask_and_scale, decode_times, decode_coords, drop_variables)
    534             k, v, concat_characters=concat_characters,
    535             mask_and_scale=mask_and_scale, decode_times=decode_times,
--> 536             stack_char_dim=stack_char_dim)
    537         if decode_coords:
    538             var_attrs = new_vars[k].attrs

/usr/lib/python3/dist-packages/xarray/conventions.py in decode_cf_variable(name, var, concat_characters, mask_and_scale, decode_times, decode_endianness, stack_char_dim)
    469         for coder in [times.CFTimedeltaCoder(),
    470                       times.CFDatetimeCoder()]:
--> 471             var = coder.decode(var, name=name)
    472 
    473     dimensions, data, attributes, encoding = (

/usr/lib/python3/dist-packages/xarray/coding/times.py in decode(self, variable, name)
    349             units = pop_to(attrs, encoding, 'units')
    350             calendar = pop_to(attrs, encoding, 'calendar')
--> 351             dtype = _decode_cf_datetime_dtype(data, units, calendar)
    352             transform = partial(
    353                 decode_cf_datetime, units=units, calendar=calendar)

/usr/lib/python3/dist-packages/xarray/coding/times.py in _decode_cf_datetime_dtype(data, units, calendar)
    105     values = indexing.ImplicitToExplicitIndexingAdapter(
    106         indexing.as_indexable(data))
--> 107     example_value = np.concatenate([first_n_items(values, 1) or [0],
    108                                     last_item(values) or [0]])
    109 

/usr/lib/python3/dist-packages/xarray/core/formatting.py in first_n_items(array, n_desired)
     92         indexer = _get_indexer_at_least_n_items(array.shape, n_desired)
     93         array = array[indexer]
---> 94     return np.asarray(array).flat[:n_desired]
     95 
     96 

/usr/local/lib/python3.6/dist-packages/numpy/core/numeric.py in asarray(a, dtype, order)
    499 
    500     """
--> 501     return array(a, dtype, copy=False, order=order)
    502 
    503 

/usr/lib/python3/dist-packages/xarray/core/indexing.py in __array__(self, dtype)
    434 
    435     def __array__(self, dtype=None):
--> 436         return np.asarray(self.array, dtype=dtype)
    437 
    438     def __getitem__(self, key):

/usr/local/lib/python3.6/dist-packages/numpy/core/numeric.py in asarray(a, dtype, order)
    499 
    500     """
--> 501     return array(a, dtype, copy=False, order=order)
    502 
    503 

/usr/lib/python3/dist-packages/xarray/core/indexing.py in __array__(self, dtype)
    492     def __array__(self, dtype=None):
    493         array = as_indexable(self.array)
--> 494         return np.asarray(array[self.key], dtype=None)
    495 
    496     def transpose(self, order):

/usr/local/lib/python3.6/dist-packages/cfgrib/cfgrib_.py in __getitem__(self, key)
     54 
     55     def __getitem__(self, key):
---> 56         return indexing.explicit_indexing_adapter(
     57             key, self.shape, indexing.IndexingSupport.OUTER, self._getitem)
     58 

AttributeError: module 'xarray.core.indexing' has no attribute 'explicit_indexing_adapter'
```

How to deal with this? 
I am usin cfgrib version 0.9.2
[icon-eu_europe_regular-lat-lon_single-level_2018050300_000_ASOB_S.zip](https://github.com/ecmwf/cfgrib/files/2509369/icon-eu_europe_regular-lat-lon_single-level_2018050300_000_ASOB_S.zip)

**Comment by alexamici** [comment]
Woops! Looks like we depend on xarray version 0.10.9. Please update with and report if this fixes your problem: `pip install xarray>=0.10.9`. Thanks!

**Comment by meteoDaniel** [comment]
@alexamici  This fixed the problem. Thanks a lot.
